docs/api/web-contents.md

```diff
@@ -1225,6 +1225,30 @@ Returns `Promise<NativeImage>` - Resolves with a [NativeImage](native-image.md)
 
 Captures a snapshot of the page within `rect`. Omitting `rect` will capture the whole visible page.
 
+#### `contents.isBeingCaptured()`
+
+Returns `Boolean` - Whether this page is being captured. It returns true when the capturer count
+is large then 0.
+
+#### `contents.incrementCapturerCount()`
```

Comment:
should be:

`contents.incrementCapturerCount(size[, stayHidden])`

Reply:
I changed it to `contents.incrementCapturerCount([size, stayHidden])` since both arguments are optional.